modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -786,6 +786,37 @@ public void tearDown() throws Exception {
 			_objectEntryLocalService, _objectRelationshipLocalService);
 	}
 
+	@Test
+	public void testAddObjectEntry() throws Exception {
```

Comment:
@vngoliveira `testAddObjectEntry` is too generic. Change the name to be more specific.